build.sbt

```diff
@@ -142,6 +142,7 @@ lazy val docs = project
     ScalaUnidoc / unidoc / unidocProjectFilter := inProjects(coreJVM, lawsJVM),
     docsMappingsAPIDir := "api",
     addMappingsToSiteDir(ScalaUnidoc / packageDoc / mappings, docsMappingsAPIDir),
+    scalacOptions := scalacOptions.value.filterNot(_ == "-Werror"),
```

Comment:
I attempted to disable fatal warnings in mdoc. It worked locally, but not in CI.